pyomo/contrib/doe/doe.py

```diff
@@ -32,17 +32,36 @@
 
 import pyomo.environ as pyo
 from pyomo.opt import SolverFactory
-import time
 import pickle
 from itertools import permutations, product
 import logging
-from pyomo.contrib.sensitivity_toolbox.sens import sensitivity_calculation, get_dsdp
-from pyomo.contrib.doe.scenario import Scenario_generator
+from enum import Enum
+from pyomo.common.timing import TicTocTimer
+from pyomo.contrib.sensitivity_toolbox.sens import get_dsdp
+from pyomo.contrib.doe.scenario import ScenarioGenerator, finite_difference_lib
 from pyomo.contrib.doe.result import FisherResults, GridSearchResult
 
+
+class calculation_mode(Enum):
```

Comment:
@jsiirola @blnicho Should this be capitalized, i.e., CalculationMode?

Reply:
@jsiirola @blnicho Any suggestions on the captialization of classes that inherit from Enum?

Reply:
Decision: CamelCase.